static/src/js/components/CollectionResults/CollectionResultsListItem.js

```diff
@@ -0,0 +1,76 @@
+import React, {
+  memo,
+  useRef,
+  useEffect
+} from 'react'
+import PropTypes from 'prop-types'
+
+import CollectionResultsItem from './CollectionResultsItem'
+
+import './CollectionResultsListItem.scss'
+
+/**
+ * Renders CollectionResultsListItem. This should be rendered as a react-window item.
+ * @param {Object} props - The props passed into the component.
+ * @param {Object} props.data - The data provided from react-window.
+ * @param {Number} props.index - The index of the item.
+ * @param {Object} props.style - The style settings for the item from react-window.
+ */
+export const CollectionResultsListItem = memo(({
+  data,
+  index,
+  style
+}) => {
+  const element = useRef()
+  const {
+    collections,
+    isItemLoaded,
+    loadMoreItems,
+    onAddProjectCollection,
+    onRemoveCollectionFromProject,
+    onViewCollectionDetails,
+    onViewCollectionGranules,
+    setSize,
+    windowWidth
+  } = data
+
+  useEffect(() => {
```

Comment:
Would it make _any_ sense to do this in breakpoint style instead of having it fire every single pixel?